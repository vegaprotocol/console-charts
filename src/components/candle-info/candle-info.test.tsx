import renderer from "react-test-renderer";

import { CandleInfo } from ".";

test("CandleInfo renders correctly", () => {
  const component = renderer.create(
    <CandleInfo
      candle={{
        date: new Date(2021, 1, 1),
        high: 100,
        low: 101,
        open: 102,
        close: 103,
        volume: 104,
      }}
      decimalPlaces={5}
    />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
