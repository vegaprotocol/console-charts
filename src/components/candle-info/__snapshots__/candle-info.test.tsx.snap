// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CandleInfo renders correctly 1`] = `
<div
  className="candle-info-wrapper"
>
  <div
    className="candle-info__item"
  >
    <span
      className="text-muted"
    >
      Candle: 
    </span>
    <span
      className="monospace-text"
    >
      00:00 01 Feb 2021
    </span>
  </div>
  <div
    className="candle-info__item"
  >
    <span
      className="text-muted"
    >
      O 
    </span>
    <span
      className="monospace-text"
    >
      102.00000
    </span>
  </div>
  <div
    className="candle-info__item"
  >
    <span
      className="text-muted"
    >
      H 
    </span>
    <span
      className="monospace-text"
    >
      100.00000
    </span>
  </div>
  <div
    className="candle-info__item"
  >
    <span
      className="text-muted"
    >
      L 
    </span>
    <span
      className="monospace-text"
    >
      101.00000
    </span>
  </div>
  <div
    className="candle-info__item"
  >
    <span
      className="text-muted"
    >
      C 
    </span>
    <span
      className="monospace-text"
    >
      103.00000
    </span>
  </div>
</div>
`;
